packages/sanity-suite/src/index-npm.ts

```diff
@@ -20,30 +20,15 @@ const getLoadOptions = (): Partial<LoadOptions> => {
   // eslint-disable-next-line sonarjs/no-all-duplicated-branches
   switch ('FEATURE' as string) {
     case 'dataResidency':
-      return {
```

Comment:
I know thatlint complains but please leave teh switch cases as is for time being (duplicated return values) as we plan to clean up separately in conjusting with out automated testing and the test rudderstack account workspace configs.

Reply:
Done.